src/service-worker.js

```diff
@@ -1,6 +1,7 @@
 import DatabaseServer from './db';
 import WebServer from './web-server';
 import docs from './docs';
+import Share from './share';
```

Comment:
All changes to this file can be undone as well (we don't need any service worker code at all now).

Reply:
`git checkout master src/service-worker.js`